packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -111,6 +118,24 @@ class SnackBarAction extends StatefulWidget {
   /// displayed in a [SnackBar].
   final VoidCallback onPressed;
 
+  /// Indicates whether the SnackBar should be hidden using the Scaffold or the
+  /// ScaffoldMessenger.
+  ///
+  /// Defaults to false in order to use the Scaffold.
+  final bool useScaffoldMessenger;
```

Comment:
Do all of the examples need to demo this property? Since it's only temporary, we should say as much here.

Reply:
Oh no, we definitely don't want anyone using this, that is why it is deprecated. Does the deprecation communicate that, or should we add more?